backend/src/api/questions.ts

```diff
@@ -27,14 +27,11 @@ export async function submitQuestionHandler(req: Request, res: Response) {
     state.type,
     userData.id,
     state.question,
-    state.answer,
     state.class,
   );
 
-  if (state.type === "Multiple Choice") {
-    for (let i = 0; i < answerList.length; i++) {
-      addAnswer(answerList[i].answer, answerList[i].correct, await questionId);
-    }
+  for (let i = 0; i < answerList.length; i++) {
+    addAnswer(answerList[i].answer, answerList[i].correct, await questionId);
```

Comment:
Might want to move `await` up to where you actually assign questionId, calling it in a loop makes it seem like we are waiting for an async action every time rather than a single value.